api/src/main/java/ai/djl/modality/cv/translator/SemanticSegmentationTranslator.java

```diff
@@ -0,0 +1,197 @@
+        ctx.setAttachment("originalWidth", image.getWidth());
+        return super.processInput(ctx, image);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public Image processOutput(TranslatorContext ctx, NDList list) {
+        // scores contains the probabilities of each pixel being a certain object
+        float[] scores = list.get(1).toFloatArray();
+
+        // get dimensions of image
+        final int width = (int) ctx.getAttachment("originalWidth");
+        final int height = (int) ctx.getAttachment("originalHeight");
+
+        // build image array
+        NDManager manager = NDManager.newBaseManager();
+        ByteBuffer bb = manager.allocateDirect(CHANNEL * height * width);
+        NDArray intRet = manager.create(bb, new Shape(CHANNEL, height, width), DataType.UINT8);
+
+        // change color of pixels in image array where objects have been detected
+        for (int j = 0; j < height; j++) {
+            for (int k = 0; k < width; k++) {
+                int maxi = 0;
+                double maxnum = -Double.MAX_VALUE;
+                for (int i = 0; i < CLASSNUM; i++) {
+
+                    // get score for each i at the k,j pixel of the image
+                    float score = scores[i * (width * height) + j * width + k];
+                    if (score > maxnum) {
+                        maxnum = score;
+                        maxi = i;
+                    }
+                }
+
+                // color pixel if object was found, otherwise leave as is (black)
+                if (maxi == PERSON || maxi == BIKE) {
+                    NDIndex index = new NDIndex(0, j, k);
+                    intRet.set(index, 0xFF00FF);
+                } else if (maxi == CAT || maxi == SHEEP || maxi == 13) {
+                    NDIndex index = new NDIndex(1, j, k);
+                    intRet.set(index, 0xFF00FF);
+                } else if (maxi == CAR || maxi == DOG) {
+                    NDIndex index = new NDIndex(2, j, k);
+                    intRet.set(index, 0xFF00FF);
+                }
+            }
+        }
+        return BufferedImageFactory.getInstance().fromNDArray(intRet);
+    }
+
+    /**
+     * Creates a builder to build a {@code SemanticSegmentationTranslator}.
+     *
+     * @return a new builder
+     */
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    /**
+     * Creates a builder to build a {@code SemanticSegmentationTranslator} with specified arguments.
+     *
+     * @param arguments arguments to specify builder options
+     * @return a new builder
+     */
+    public static Builder builder(Map<String, ?> arguments) {
+        Builder builder = new Builder();
+
+        builder.configPreProcess(arguments);
+        builder.configPostProcess(arguments);
+
+        return builder;
+    }
+
+    /** Resizes the image based on the shorter edge or maximum edge length. */
+    private class ResizeShort implements Transform {
```

Comment:
```suggestion
    private static final class ResizeShort implements Transform {
```

Reply:
The ResizeShort class from the [InstanceSegmentationTranslator](https://github.com/deepjavalibrary/djl/blob/a1c74485310933cb99ea0952b43a8dec4c59cef1/api/src/main/java/ai/djl/modality/cv/translator/InstanceSegmentationTranslator.java#L152) is not static or final, and by making it static, the `shortEdge`, `maxEdge`, `rescaledHeight`, and `rescaledWidth` variables must be initialized at the beginning of the file. If I make it static, what values should they be assigned?